fast64_internal/f3d/f3d_gbi.py

```diff
@@ -3000,45 +2923,7 @@ def to_binary(self):
         return bytearray(self.color + [0x00] + self.color + [0x00] + self.normal + [0x00] + [0x00] * 4)
 
     def to_c(self):
-        return ",".join(
-            (
-                "0x" + format(self.color[0], "X"),
-                "0x" + format(self.color[1], "X"),
-                "0x" + format(self.color[2], "X"),
-                "0x" + format(self.normal[0], "X"),
-                "0x" + format(self.normal[1], "X"),
-                "0x" + format(self.normal[2], "X"),
-            )
-        )
-
-    def to_sm64_decomp_s(self):
-        return (
-            ".byte "
-            + ", ".join(
-                (
-                    "0x" + format(self.color[0], "X"),
-                    "0x" + format(self.color[1], "X"),
-                    "0x" + format(self.color[2], "X"),
-                    "0x00",
-                    "0x" + format(self.color[0], "X"),
-                    "0x" + format(self.color[1], "X"),
-                    "0x" + format(self.color[2], "X"),
-                    "0x00",
-                )
-            )
-            + "\n"
-            + ".byte "
-            + ", ".join(
-                (
-                    "0x" + format(self.normal[0], "X"),
-                    "0x" + format(self.normal[1], "X"),
-                    "0x" + format(self.normal[2], "X"),
-                    "0x00",
-                    "0x00, 0x00, 0x00, 0x00",
-                )
-            )
-            + "\n"
-        )
+        return ', '.join([f"0x{a:X}" for a in (*self.color, *self.normal)])
```

Comment:
please run the black formatter on this file
idk if it's the only thing but use of `'` tells me it needs to be

Reply:
I actually forgot to mention this myself - there's a couple formatting differences in this file when running black formatter on it.